spec.html

```diff
@@ -5893,15 +5903,16 @@
 
     <!-- es6num="8.2.3" -->
     <emu-clause id="sec-setrealmglobalobject" aoid="SetRealmGlobalObject">
-      <h1>SetRealmGlobalObject ( _realmRec_, _globalObj_ )</h1>
-      <p>The abstract operation SetRealmGlobalObject with arguments _realmRec_ and _globalObj_ performs the following steps:</p>
+      <h1>SetRealmGlobalObject ( _realmRec_, _globalObj_, _thisValue_ )</h1>
+      <p>The abstract operation SetRealmGlobalObject with arguments _realmRec_, _globalObj_, and _thisValue_ performs the following steps:</p>
```

Comment:
This is because it's quite hard to get ES-style compositional default values in ECMASpeak. If we stuck with "not passed," the result would be that InitializeHostDefinedRealm() would have to branch and call out to either SetRealmGlobalObject(realmRec, globalObj) or SetRealmGlobalObject(realmRec, globalObj, thisValue). In other words, **undefined** and not passed are not currently equivalent in ECMASpeak.